### ISSUE TITLE
Image crop editor should not be limited to scale width/height directly

### ISSUE THREAD

**Issue by drakulis** [sulu/sulu#5557]
| Q | A
| --- | ---
| Sulu Version | 2.1.2
| Browser Version | All

# Actual Behavior
I have defined following format:
```
    <format key="x400">
        <meta>
            <title lang="pl">Baner h400</title>
            <title lang="en">Banner h400</title>
            <title lang="de">Banner h400</title>
        </meta>

        <scale y="400"/>
    </format>
```
Media crop tool use `y` attribute of `scale` tag of the image format to limit height in the editor.

![sulu](https://user-images.githubusercontent.com/1709738/96453023-31453180-121a-11eb-94df-59b6039343c3.png)

**The problem is even more serious when you are working on small resolution (or just resize browser window) then image editor height is less then `y` attribute and i cannot crop the image at all.**
# Expected Behavior

Media crop tool should not limit the height of the editor to `y` attribute of the image format.

# Steps to Reproduce

- define image format as described above
- upload high resolution image eg. 1920x1080
- resize browser window to 1280×720
- try to crop image to 400 pixels from the top of the original image
RESULT: you cannot crop image height to 400px even if original image is 1080px height. 
What you get is 400px of scaled image in the editor.

**Comment by alexander-schranz** [comment]
@drakulis Thank you for reporting this issue. Can you also provide the original image you try to crop? As the calculation depends also from the original image size and with my test image I could not reproduce it.

**Comment by nnatter** [comment]
This sounds like a problem that should be fixed with #5399. The fix was included in 2.1.1. Is it possible that your project uses an old build of the administration interface? See `Step 4` of the `Upgrading Sulu 2.x`

**Comment by drakulis** [comment]
Just tested it on latest SULU 2.2.0RC1 and it seems to be fixed there. Closing the issue.

@nnatter its possible that I was using outdated admin panel
